packages/vue/src/components/CcvWordCloudChart.ts

```diff
@@ -1,4 +1,4 @@
-import { WordCloudChart, type WorldCloudChartOptions } from '@carbon/charts'
+import { WordCloudChart, type WordCloudChartOptions } from '@carbon/charts'
 import { chartFactory } from './chartFactory'
 
-export default chartFactory<WorldCloudChartOptions>(WordCloudChart, 'CcvWordCloudChart')
+export default chartFactory<WordCloudChartOptions>(WordCloudChart, 'CcvWordCloudChart')
```

Comment:
we should still export it as world as well and mark as deprecated

Reply:
Updated